### ISSUE TITLE
3.15 Changelog

### ISSUE THREAD

**Issue by stackpoint** [Thalassicus/cep-bnw#193]
Armies
- Samurai strength reduced from 37 to 28.

Cities
- Sistine Chapel should now correctly grant culture boosts.
- Hagia Sophia should now correctly grant faith from specialists.
- Wat Phra Kaew should now correctly grant +10% modifiers to shrines/temples.
- Stele shoulud now correctly grant +1 faith per population.
- Churches of Lalibela now grants a Missionary with 3 spreads

Interface
- City View and Top Panel should be synced up with the reactivated YieldLibrary.
- Top Panel should now have several new features (from GEM).
- Disabled ingame restart button (was causing caching bugs).

Leaders
- Allotment Trait should now spawn all appropriate land and air units.
- Arabia should now display the correct text for its unique trait.
- Zulu trait text should now state that vanguard units have a -50% maintenance cost

Policies
- Piety Opener: Gives +1 culture, +1 faith, and double production for shrines.
- Unity Policy: Now gives 50% cost discount for Golden Ages
- Meditation Policy: Now gives 33% Faith Cost bonus
- Charity Policy: Now gives 2 gold from Temples
- Piety Finisher: No longer gives culture from Holy Sites
- Wealth Opener: Reduced gold and production in all cities from 15% to 10%
- Caravans: Increased gold from +2 to +3 per trade route
- Entrepreneurship: Added +1 free Great Merchant
- Protectionism: Removed +10% gold and production
- Maritime Infrastructure: Gives 4 production per Coastal City and 4 gold per Lighthouse, Harbor, Seaport
- Colonization: Cities start with three popuplation. 2 Settlers, 2 Workers, and 2 Defense Units appear at the Capital
- Republic: Now gives +1 production from defense buildings
- Discipline: Combat bonus now affects all units
- Gladiators:Now gives 2 defense units 
- Professional Army: Helps produce all types of military units
- Removed unintentional happiness and policy discount from Representation
- Fixed unintentional production modifier for military buildings from Professional Army
- Fixed Humanism unintentionally giving happiness from Research labs
- Philantropy policy changes if the City State Diplomacy mod is used
- Tradition now unlocks Banaue Rice Terraces
- Tradition gives 1 Food per population
- Spoils of War gives Gold to value of 3x the enemy Strength value
- CounterIntelligence doubles anti-spy building production
- Secularism increases Science from specialists

Religion
- Feed The World belief now gives two food from buildings

Research
- Archery now a prerequisite for The Wheel instead of a prerequisite for Mathematics.

Other
- CEP now automatically checks if CSD is used (entry from CAT options file removed)
- Should now be compatible with the non-DLL versions of the City State Diplomacy (CSD) mod.
- Improved compatibility with WHoward's DLL mod
- Heal on promotion now occurs on button click (instead of end of turn)

**Comment by stackpoint** [comment]
The changelog of the next release would normally be made as a draft in the releases section. However, this time around nobody has drafted the general release notes as we were making changes. 

@GrantSP, @skodkim, @EricB1: Please follow the formatting of the original post to include any commits you have made to either branches for the next release. I'll add your comments to the original post.

**Comment by stackpoint** [comment]
Please don't forget to modify the [Google Spreadsheets](https://drive.google.com/?authuser=0#folders/0B58Ehya83q19eVlvWXdmLXZ6UUE) with any changes you have made that is generated from them (text, units, tech).

Also, if you haven't made the corresponding text changes to any policy modifications you may have made, I recommend you go ahead and do so now.

**Comment by EricB1** [comment]
Here's a summary of changes that I have made:

Tech
Archery now a prerequisite for The Wheel instead of a prereq for Mathematics

Leaders
Fixed a bug with Arabia not displaying the correct text for its unique trait.
Japan - Samurai strength reduced from 37 to 28

Policies
Piety Opener gives +1 culture, +1 faith, and double production for shrines.
Wealth Opener: reduced gold and production in all cities from 15% to 10%
Caravans: increased gold from +2 to +3 per trade route
Entrepreneurship: added +1 free Great Merchant
Protectionism: removed +10% gold and production
Maritime Infrastructure: added Harbors to receive +2 gold

**Comment by stackpoint** [comment]
Added. Thanks!

**Comment by skodkim** [comment]
Just put in my changes. Are we working with an ETA on the release?

\Skodkim

**Comment by skodkim** [comment]
@EricB1 Would you update the OP of the Policy updates thread with you changes https://github.com/Thalassicus/cep-bnw/issues/167 as well?

Seems we're pretty far with this. I keep looking at them but I seem to always conclude that the remaining are beyond my skills :unamused:

\Skodkim

**Comment by stackpoint** [comment]
@Thalassicus would have to confirm, but if I had to guess, it would be either this weekend or the next after since I'm almost finished with the top panel modifications too.

**Comment by EricB1** [comment]
@stackpoint 
Here's another one.

Leaders - corrected Zulu trait text to state that vanguard units also have a -50% maintenance cost

**Comment by stackpoint** [comment]
@GrantSP, usually when you fix a bug without adding features, you should still increase the revision number: 3.14.1 -> 3.14.2. It's called [semantic versioning](http://semver.org). Because incrementing a number and making another release post is less of a pain for everybody than the users not having the latest version of the mod even if the tag on it says they do.

**Comment by stackpoint** [comment]
Also the very long and technical post you made for the release page regarding the translations changes should be instead on a wiki page titled "Translations" or something of the sort and should only be referenced on the release page because that's a lot of information not very useful to many of the users upon download versus information such as a changelog.

**Comment by Thalassicus** [comment]
I'm getting involved in modding again. I worked on this a lot last week, and I'm planning to release the update Wednesday.

Are the notes on the release page hidden? I thought drafts show up for everyone. Here's the draft I started a few months ago: https://github.com/Thalassicus/cep-bnw/releases/tag/untagged-48a01944f89de77531d4

I like to organize release notes like this:
- Brief summary of biggest new features in a few sentences.
- Sections for individual feature changes.
- Bugfixes.

I patterned this on how successful game studios organize their notes. The main sections show changes in intended features. Bug-fixes only get things back to the intended state, so they go together in a less-emphasized group at the end.
